pkg/kubectl/kubectl.go

```diff
@@ -197,6 +197,20 @@ func ResourceShortFormFor(resource string) (string, bool) {
 	return alias, exists
 }
 
+// ResourceLongFormFor looks up for a long form of resource names.
+func ResourceLongFormFor(resource string) (string, bool) {
```

Comment:
I think this would be more correct as a check using the `RESTMapper` we use since it includes the shortcut expander and this list will be legacy after we complete https://github.com/kubernetes/kubernetes/pull/40174 and related.

Also this doesn't handle the group correctly.